### PR TITLE
Check the bundle URL for CSP while loading urn:uuid resources

### PR DESCRIPTION
**Pull request by horo-t**
I created this pull request on @hayatoito 's behalf (PR https://github.com/WICG/webpackage/pull/653 for Issue https://github.com/WICG/webpackage/issues/651), because he is busy on other tasks.

Here is the Chromium side CL: https://crrev.com/c/2886721

### CODE REVIEW

File: explainers/subresource-loading-opaque-origin-iframes.md [hunk 1]
```diff
@@ -65,36 +65,43 @@ Note:
...
+<script src="urn:uuid:429fcc4e-0696-4bad-b099-ee9175f023ae"></script>
 <iframe src="urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6"></iframe>
 ```
 
+Note:
+- Loading `urn:uuid` resources from web bundles served from HTTPS server is
+  allowed when "\*" is set in the CSP
+  [source expression](https://w3c.github.io/webappsec-csp/#source-expression).
+  This is different from the CSP behavior that `data:` and `blob:` schemes are
+  excluded from matching a policy of "\*".
```
Comment (kinu) [inline explainers/subresource-loading-opaque-origin-iframes.md:102]: Could we also write why (extract some relevant part from the issue about) why it is okay for urn:uuid (contrary to data: and blob: where it's considered not okay) to do this?

Comment (horo-t) [reply explainers/subresource-loading-opaque-origin-iframes.md]: Done. How about this?
https://github.com/WICG/webpackage/pull/654/commits/4f06ad3d824c8c277cd5161bb7d9074c628de4b2

**Comment by horo-t** [comment]
@hayatoito @kinu 
Ping?

**Review by hayatoito — APPROVED** [review]
LGTM

**Review by kinu — APPROVED** [review]
Sorry for the delay, lgtm!

**Comment by horo-t** [comment]
@jyasskin 
Could you please review this PR if you have time?

**Review by jyasskin — APPROVED** [review]
Sorry for the slow review. This looks good.

**Pull request merged**